include/exadg/structure/spatial_discretization/operator.cpp

```diff
@@ -304,6 +304,21 @@ Operator<dim, Number>::setup_operators()
   operator_data.material_descriptor = material_descriptor;
   operator_data.unsteady            = (param.problem_type == ProblemType::Unsteady);
   operator_data.density             = param.density;
+
```

Comment:
The structure module has no Picard yet. The nonlinear solver requires the according data structure and the rhs for Picard nonetheless. Both are asserted for 'NonlinearSolver::SolverType::Picard'.